### PR TITLE
defect (WithTable): Fix WithVersion(...) to work with WithTable(...)

### PR DESCRIPTION
**Pull request by jimlambrt**
Fixes issue #9: WithTable(...) not working correctly for when used in
addition to WithVersion(...

### CODE REVIEW

File: rw.go [hunk 1]
```diff
@@ -147,7 +147,20 @@ func (rw *RW) whereClausesFromOpts(_ context.Context, i interface{}, opts Option
...
+			var tableName string
+			switch {
+			case opts.WithTable != "":
+				tableName = opts.WithTable
+			default:
+				tableName = mDb.Statement.Schema.Table
+			}
+			where = append(where, fmt.Sprintf("%s.version = ?", tableName)) // we need to include the table name because of "on conflict" use cases
+		} else {
+			where = append(where, "version = ?") // we need to include the table name because of "on conflict" use cases
```
Comment (louisruch) [inline rw.go:162]: Comment needs to be updated, or removed

**Review by louisruch — COMMENTED** [review]
Two quick comments otherwise this looks good

File: rw.go (hunk 1 above)
Comment (jimlambrt) [reply rw.go:162]: ty!

File: rw_unexported_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	tests := []struct {
+		name      string
+		rw        *RW
+		i         interface{}
+		opts      Options
+		wantWhere string
+		wantArgs  []interface{}
+		wantErr   bool
+	}{
+		{
```
[the remark this answers is not in the document]
Comment (jimlambrt) [reply rw_unexported_test.go:27]: is that the next test?

**Review by louisruch — APPROVED** [review]
LGTM

**Pull request merged**